stripe/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowControllerInitializer.kt

```diff
@@ -112,13 +107,19 @@ internal class DefaultFlowControllerInitializer(
                         PaymentMethod.Type.fromCode(it)
                     }
 
+                val savedSelection = if (isGooglePayReady) {
```

Comment:
inline? as it's not used else where

Reply:
it just looks so pretty right now with all the parameters :D